packages/site-kit/src/lib/docs/DocsOnThisPage.svelte

```diff
@@ -299,9 +312,11 @@
 			padding: 0.5rem;
 
 			border-radius: var(--sk-border-radius);
-			box-shadow: 0px 0px 14px rgba(0, 0, 0, 0.1);
+			/* box-shadow: 0px 0px 14px rgba(0, 0, 0, 0.1); */
```

Comment:
we should remove commented out styles before merging

Reply:
Done